views/code.go

```diff
@@ -0,0 +1,83 @@
+package views
```

Comment:
I feel this package should be called `models` instead of `views` . Wdty?

Reply:
This is what I'm referring to btw.
https://github.com/devguyio/kuby/blob/2487c589961359ee25dee7f766f922eebf5dd147/views/listPods.go#L83-L125

In the latest commit I refactored this a bit to make testing and mocking slightly easier.
[https://github.com/piotrpdev/kuby/blob/2487c589961359ee25dee7f766f922eebf5dd147/views/listPods.go#L68-L125](https://github.com/piotrpdev/kuby/blob/2487c589961359ee25dee7f766f922eebf5dd147/views/listPods.go#L68-L125)
![image](https://github.com/devguyio/kuby/assets/99439005/3f20d7ba-176a-480d-8af8-3768998517df)

Reply:
> I also thought about that, I can rename it to that but it has more than just the model. It also has the code for turning the slice of a resource e.g. pods to a slice of rows that is displayed in the UI. I previously had this in a separate package, but I thought it was simpler to have all the code to do with pods all in one module.

I think it's still fine to call it `model` because that's a code that's kinda relevant to the models. And it'd make more sense to have a package name that represent the main added value this package provides when you import it.